### ISSUE TITLE
Feature: Convert YAML into JSON

### ISSUE THREAD

**Issue by rafi** [yokattana/json-yaml#1]
It could be really awesome for this tool to convert the other way around too, YAML to JSON.

**Comment by yokattana** [comment]
Agreed, that would be a useful feature. I can't commit to implementing it soon but I'll keep it in mind.
